README.md

```diff
@@ -1,5 +1,180 @@
+- `WWP_seq_new_run_auto.sh`
+- `run_viralrecon.sh`
+- `run_freyja.sh`
+
+## Recommended Usage
+
+To run all the steps of the analysis in one single script. This is the recommended way of running the wastewater genomics analysis unless you need to troubleshoot the individual scripts when this one fails. Remember to start a screen as this process can take upto hours depending on the sequence data generated.
+
+Execute the bash script with the sequencing `run_name` as an argument:
+
+```bash
+sh run_wwtp_sequencing_analysis.sh <wastewater sequencing run_name>
+```
+Here are the steps involved in executing the analysis in more detail. Please note that you won't need to run these scripts individually for routine data analysis.
+
+### Step 1: Execute WWP_seq_new_run_auto.sh
+Execute the bash script with the sequencing `run_name` as an argument:
+
+```bash
+sh WWP_seq_new_run_auto.sh <wastewater sequencing run_name> | tee -a WWP_seq_new_run_auto.log
+```
+
+This script initializes the sequencing run analysis. It checks whether the fastq generation step is completed on the in-house DRagen server, sets up the required directory structure for analysis, moves any failed samples to a dedicated directory, renames fastq files for downstream analysis, and prepares fastq files for NCBI submission.
+
+This script also generates a csv file with NCBI submission ID and associated fastq file names which are used for generating NCBI submission templates in Data-flo.
+
+### Step 2: Execute run_viralrecon.sh
+Execute the bash script with the sequencing `run_name` as an argument:
+
+```bash
+sh run_viralrecon.sh <wastewater sequencing run_name> | tee -a viralrecon.log
+```
+
+This script executes the [Viralrecon](https://github.com/nf-core/viralrecon) pipeline with wastewater sequencing data. It checks and creates an input samplesheet required to run the Viralrecon pipeline, runs the pipeline, and copies the resulting files to a dedicated `results` directory.
+
+This script also handles post-pipeline cleanup by removing the `work` directory.
+
+### Step 3: Execute run_freyja.sh
+Execute the bash script with the sequencing `run_name` as an argument:
+
+```bash
+sh run_freyja.sh <wastewater sequencing run_name> | tee -a freyja.log
+```
+
+This script retrieves BAM files for each sample generated by the Viralrecon tool and performs [Freyja](https://github.com/andersen-lab/Freyja/tree/main) analysis. It uses BAM files after the ivar primer trimming step and generates Freyja demultiplexed lineage data. It then aggregates the lineage data and creates a lineage plot. Finally, it copies the aggregated lineage results to the `results` directory.
+
+## Note
+
+You may need to adjust the `SINGULARITY_CACHEDIR` and `NXF_SINGULARITY_CACHEDIR` environment variables according to your system configuration in the `run_viralrecon.sh` script.
+
+Local copy of this github repo is located at /Volumes/NGS/Bioinformatics/pooja/ww_analysis_scripts/Wastewater-genomic-analysis.
```

Comment:
When it's ready for production, we can move it to /home/Bioinformatics/Wastewater-genomic-analysis to protect it from edits